lib/SearchQueue.js

```diff
@@ -19,11 +19,28 @@ SearchQueue.prototype = {
      var key = snap.key;
 
      if (this._assertValidSearch(key, dat)) {
-      // Perform (a very simple) ElasticSearch query
-      this.esc.search({
-        index: dat.index,
-        q: dat.query
-      }, function(error, response) {
+      // Perform ElasticSearch query
```

Comment:
Please see my recent changes, which conflict with this edit. Also, I don't want to restrict the format of the query/body parameter. There are tons upon tons of possible options/structures users can put into body and I'd rather not get into the business of enforcing ES schemas.